apps/api/src/auth/guest_auth.py

```diff
@@ -88,7 +88,9 @@ async def _get_existing_key(email: EmailStr) -> Optional[str]:
     # Reject expired key
     now = utc_now()
     if now > auth.exp:
-        await _remove_guest_key(uid)
+        await mongodb_handler.update_one(
+            Collection.USERS, {"_id": uid}, {"guest_auth": None}
+        )
```

Comment:
Could we extract this database call to another named function such as `_remove_expired_guest_key`?